Annotato/Annotato/Views/Document/List/DocumentListViewController.swift

```diff
@@ -43,8 +43,25 @@ class DocumentListViewController: UIViewController {
     }
 }
 
-extension DocumentListViewController: DocumentListToolbarDelegate, Navigable {
+extension DocumentListViewController: DocumentListToolbarDelegate, UIDocumentPickerDelegate, Navigable {
+
     func didTapAddButton() {
         goToDocumentEdit()
     }
+
+    func didTapImportFileButton() {
+        goToImportingFiles()
+    }
+
+    func documentPicker(_ controller: UIDocumentPickerViewController, didPickDocumentsAt urls: [URL]) {
+        guard let selectedFileUrl = urls.first else {
+            return
+        }
+        guard let dir = FileManager.default.urls(for: .documentDirectory, in: .userDomainMask).first else {
+            return
+        }
+        print("selected file url = \(selectedFileUrl)")
+        let newlyLoadedDocumentPdfViewModel = DocumentPdfViewModel(url: selectedFileUrl)
+        goToDocumentEdit(documentPdfViewModel: newlyLoadedDocumentPdfViewModel)
```

Comment:
Since (I think?) a document should not exist without a pdf, we either let them
1. Import a pdf first, then create a document from it --> complete remove the add button
2. Click the add button first to go to a document edit view, then import the pdf file when they are there. If no pdf is imported yet, they cannot make any annotation and it won't be saved.

Otherwise right now the add button isn't correct? We are just hardcoding a sample file there but it's not integrated with the rest

Aka there is no need to have 2 buttons.